programs/ssl/ssl_server2.c

```diff
@@ -659,6 +661,7 @@ sni_entry *sni_parse( char *sni_string )
                 goto error;
         }
 
+#if defined(MBEDTLS_X509_CRL_PARSE_C)
```

Comment:
Minor: Should we print an error message if the CRL parameter is not `-` but `MBEDTLS_X509_CRL_PARSE_C` isn't set?

Reply:
Error out: yes, ok, I'll amend the commit. Print a message: out of scope since none of the other errors in this function trigger an error message.